test-case/check-audio-equalizer.sh

```diff
@@ -49,8 +50,8 @@ func_test_eq()
     local id=$1
     local conf=$2
 
-    dlogc "sof-ctl -Dhw:$sofcard -n $id -s $conf"
-    sof-ctl -Dhw:"$sofcard" -n "$id" -s "$conf" || {
+    dlogc "sof-ctl -Dhw:$sofcard -c name='$id' -s $conf"
+    sof-ctl -Dhw:"$sofcard" -c name=\""$id"\"  -s "$conf" || {
```

Comment:
This gives a quoted ID to `sof-ctl` itself. Does `sof-ctl` really needs to see double quotes and why? This deserves at least a comment IMHO.

If this is really what you want, you could also make it more explicit with an intermediate variable:
```shell
  local double_quoted_id=\""$id"\"

       ... -c name="$double_quoted_id"
```